src/app/component/pages/participant/participant-session/paricipant-session.component.html

```diff
@@ -0,0 +1,10 @@
+<app-header>
+  <span (click)="onClickLogout()" aria-label="Logout">
+    logout
+  </span>
+</app-header>
+<app-content>
+  <div *ngIf="sessionId">
+    <p>active session</p>
+  </div>
+</app-content>
```

Comment:
Komponente umbenennen in participant mit "t"

Reply:
Wird mit dem Ticket LFD-40 upgedatet.